ksql-rest-app/src/main/java/io/confluent/ksql/rest/server/computation/CommandStore.java

```diff
@@ -1,12 +1,12 @@
 /**
  * Copyright 2017 Confluent Inc.
- *
+ * <p>
```

Comment:
If we add <p> here we need to update all the files for consistency. All other confluent project also don't have <p> in the license text. Let's keep the license text consistent with all other confluent projects.